src/containers/Tenant/QueryEditor/QueryExplain/QueryExplain.js

```diff
@@ -102,12 +102,6 @@ function QueryExplain(props) {
         };
     }, []);
 
-    useEffect(() => {
```

Comment:
It occurred, that `explain-ast` is run in data-query mode, so when we request `ast` for `explain-scan` or `explain-query`, we have `ast` that does not correspond to initial mode. Thus, it should be displayed only if query response have it